packages/build/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "27.18.4",
   "description": "Netlify build module",
   "type": "module",
-  "exports": "./src/core/main.js",
-  "main": "./src/core/main.js",
+  "exports": "./lib/core/main.js",
+  "main": "./lib/core/main.js",
   "types": "types/index.d.ts",
   "bin": {
-    "netlify-build": "./src/core/bin.js"
+    "netlify-build": "./lib/core/bin.js"
   },
   "files": [
-    "src/**/*.js",
-    "src/**/*.yml",
+    "lib/**/*.js",
+    "lib/**/*.yml",
```

Comment:
are you copying those files as well? I don't see that please diff the package outputs from main branch and your branch and check if the published packages are the same

Reply:
> please fix the package lock file regeneration 

I don't understand what you mean here, the lock file was updated as a result of adding lerna & nx. My guess is the diff is large because of their peer deps.

Reply:
If safety is important, I will revert back to npm workspaces.

Reply:
I diffed the output of this & main. There's no diff.